config.js.docker

```diff
@@ -8,6 +8,29 @@ function getParam(name, _default) {
   return argv[name] || process.env[envname] || _default;
 }
 
+function getIntParam(name, _default) {
+  const param = getParam(name, _default);
+  const parsedParam = parseInt(param);
+  if (Number.isInteger(parsedParam)) {
+    return parsedParam;
+  } else {
+    return _default;
+  }
+}
+
+function getBooleanParam(name, _default) {
+  const param = getParam(name, _default);
+  if (param === 'true' || param === '1') {
+    // Should we allow more representations?
+    return true;
+  } else if (param === 'false' || param === '0') {
+    // Should we allow more representations?
+    return false;
+  } else {
+    return _default;
```

Comment:
I'd say throw an exception to prevent running with wrong parameters.

Reply:
If this is an optional parameter we can't accept only 'true', 'false', '0' and '1'. We must accept the default parameter, which is `undefined` or `null`. That's why I kept the `else`

Reply:
But you also return the default value for any random string (e.g. "a").

Reply:
Yeah, I've done the same validation for the integer and boolean methods. Can you see this commit? https://github.com/HathorNetwork/hathor-wallet-headless/pull/82/commits/212631446fdf39a2d87ca54e2c499cbc9884b8c0